README.md

```diff
@@ -150,6 +150,23 @@ Other Style Guides
     console.log(a); // ReferenceError
     console.log(b); // ReferenceError
     ```
+  <a name="references--difference-let-var"></a><a name="2.4"></a>
+  - [2.4](#references--difference-let-var) Difference between `let` and `const`
+    
+    For loop with **var** keyword
```

Comment:
In the JS variable var is define in a scope local. we can recall it in a global scope.

Only in JS we can do it.

Variable let is working only in local scope.

Reply:
I'm not sure what this comment means. `var`, `let`, and `const` are all local scope - var is function scope, and the other two are block scope. None of them are globally scoped.